### DISCUSSION TITLE
vue-ts build error

### DISCUSSION THREAD

**Discussion by ghost** [element-plus/element-plus#1817] [Q&A]
node_modules/element-plus/lib/el-cascader-panel/src/node.d.ts:4:20 - error TS2304: Cannot find name 'Nullable'.

4     readonly data: Nullable<CascaderOption>;

node_modules/element-plus/lib/el-form/src/token.d.ts:28:11 - error TS2304: Cannot find name 'ComponentSize'.

28     size: ComponentSize;

**Comment by hyphappy011** [comment]
I met this issue too

**Comment by hyphappy011** [comment]
node_modules/element-plus/lib/el-cascader-panel/src/node.d.ts:4:20 - error TS2304: Cannot find name 'Nullable'.

4     readonly data: Nullable<CascaderOption>;
                     ~~~~~~~~

node_modules/element-plus/lib/el-cascader-panel/src/node.d.ts:22:23 - error TS2304: Cannot find name 'Nullable'.

22     constructor(data: Nullable<CascaderOption>, config: CascaderConfig, parent?: Node, root?: boolean);
                         ~~~~~~~~

node_modules/element-plus/lib/el-cascader-panel/src/types.d.ts:50:20 - error TS2304: Cannot find name 'Nullable'.

50     expandingNode: Nullable<CascaderNode>;
                      ~~~~~~~~

node_modules/element-plus/lib/el-form/src/token.d.ts:28:11 - error TS2304: Cannot find name 'ComponentSize'.
